client-api/src/main/java/org/sonarsource/sonarlint/core/client/api/util/FileUtils.java

```diff
@@ -39,13 +42,54 @@ private FileUtils() {
     // utility class, forbidden constructor
   }
 
-  public static void moveDir(Path temp, Path dest) {
+  public static void moveDir(Path src, Path dest) {
     try {
-      Files.move(temp, dest, StandardCopyOption.ATOMIC_MOVE);
-    } catch (AtomicMoveNotSupportedException e) {
-      throw new IllegalStateException("Atomic move not supported for " + temp + " to " + dest, e);
+      try {
+        Files.move(src, dest, StandardCopyOption.ATOMIC_MOVE);
+      } catch (AtomicMoveNotSupportedException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1166)